src/api/app/models/event/request.rb

```diff
@@ -84,10 +84,20 @@ def source_maintainers
       action_maintainers('sourceproject', 'sourcepackage')
     end
 
+    def source_project_watchers
+      source_watchers
+    end
+
+    def target_project_watchers
+      target_watchers
+    end
```

Comment:
Shouldn't this be the opposite? So the old method (`source_watchers`) calls the new one ( `source_project_watchers`)? As you did before with `watchers`.

The same would apply for `target_watcher`.

Reply:
i could do it the other way around as well, works both. but yeah then we just have to remove the old method instead of transfering the code. I will adapt

Reply:
done :)